students/hwayeon/users/views.py

```diff
@@ -33,5 +33,21 @@ def post(self, request):
 
             return JsonResponse({"message": "SUCCESS"}, status=201)
 
+        except KeyError:
+            return JsonResponse({"message": "Key_Error"}, status=400)
+
+class SigninView(View):
+    def post(self, request):
+        try:
+            data         = json.loads(request.body)
+            email        = data['email']
+            password     = data['password']
+
+            if not User.objects.filter(email = data['email'], password = data['password']).exists():
```

Comment:
이미 line 43, 44에서 email 변수를 선언해 주었는데 line 46에서 다시 `data["email"]`, `data["password"]`를 사용해서 dictionary에서 꺼내오고 있네요!

이 경우라고 하면 line 43, 44 코드를 지워도 될 것 같습니다.

```suggestion
            if not User.objects.filter(email = data['email'], password = data['password']).exists():
```

Reply:
수정 완료하였습니다 :)